templates/main_template.yml

```diff
@@ -42,8 +42,7 @@ jobs:
 
       - name: Zip output
         run: |
-          cd plugins
-          zip -qq -r ${plugin_name}-${{ steps.setup_sp.outputs.plugin-version }}.zip ${plugin_name}.smx
+          zip -qq -y -r ${{ github.event.repository.name }}-${{ steps.setup_sp.outputs.plugin-version }}.zip configs plugins scripting extensions gamedata translations data
```

Comment:
Plugins generally require or have multiple files, for example a gamedata file or some translation files. There is no reason to zip only the plugin file. This zips everything even if certain folders don't exist.